crates/sozo/ops/src/statistics.rs

```diff
@@ -1,45 +1,83 @@
 use std::fs::{self, File};
-use std::io::{self, BufReader};
+use std::io::{self, BufReader, Seek, SeekFrom};
 use std::path::PathBuf;
 
 use anyhow::{Context, Result};
+use cairo_lang_starknet_classes::casm_contract_class::CasmContractClass;
+use cairo_lang_starknet_classes::contract_class::ContractClass;
 use camino::Utf8PathBuf;
+use scarb_ui::Ui;
 use starknet::core::types::contract::SierraClass;
 use starknet::core::types::FlattenedSierraClass;
 
 #[derive(Debug, PartialEq)]
 pub struct ContractStatistics {
     pub contract_name: String,
-    pub number_felts: u64,
-    pub file_size: u64,
+
+    pub sierra_bytecode_size: u64,
+    pub sierra_contract_class_size: u64,
+
+    pub casm_bytecode_size: u64,
+    pub casm_contract_class_size: u64,
 }
 
-fn read_sierra_json_program(file: &File) -> Result<FlattenedSierraClass> {
-    let contract_artifact: SierraClass = serde_json::from_reader(BufReader::new(file))?;
+fn read_sierra_json_program(file: &File) -> Result<(FlattenedSierraClass, CasmContractClass)> {
```

Comment:
We may change the function name as it's doing a bit more than mentioned. But can keep this name for now if not too confusing for you.